components/Nav.js

```diff
@@ -1,36 +1,37 @@
-import { html, react, css } from 'https://unpkg.com/rplus-production@1.0.0';
+import { html, css } from 'https://unpkg.com/rplus-production@1.0.0';
 
 import { RadioGroup } from './RadioGroup.js';
 import { PackageOverview } from './PackageOverview.js';
 import { RegistryOverview } from './RegistryOverview.js';
 import { FileOverview } from './FileOverview.js';
-const isEmpty = obj => Object.keys(obj).length === 0;
+import { useStateValue } from '../utils/globalState.js';
 
-export default ({ file, versions, dispatch }) => {
-  const [mode, setMode] = react.useState('package');
+export default () => {
+  const [{ mode }, dispatch] = useStateValue();
   const modeOptions = {
     registry: mode === 'registry',
     package: mode === 'package',
     file: mode === 'file',
   };
-
   return html`
     <nav className=${styles}>
-      <${RadioGroup} options=${modeOptions} onClick=${setMode} />
-      ${!isEmpty(file) &&
-        (mode === 'package'
-          ? html`
-              <${PackageOverview} file=${file} versions=${versions} />
-            `
-          : mode === 'registry'
-          ? html`
-              <${RegistryOverview} />
-            `
-          : mode === 'file'
-          ? html`
-              <${FileOverview} file=${file} dispatch=${dispatch} />
-            `
-          : null)}
+      <${RadioGroup}
+        options=${modeOptions}
+        onClick=${val => dispatch({ type: 'setMode', payload: val })}
+      />
+      ${mode === 'package'
+        ? html`
```

Comment:
I would prefer this to be a switch but also I'm the only member of the switch lobby so 🤷‍♀